example/freight_service_test.go

```diff
@@ -54,7 +54,7 @@ func Test_FreightService(t *testing.T) {
 
 func Test_FreightService_AlternativeSetup(t *testing.T) {
```

Comment:
IMO could be worth running both alternative setups, i.e. both `examplefreightv1 .TestFreightService` and `examplefreightv1.TestServices`, at least to make sure these are not silently broken in any future commits 😇

Reply:
Great that you noticed that, I wonder if we should make the specific service test private so you only have one test function `examplefreightv1.TestServices`, since if you use `examplefreightv1 .TestFreightService`, you might miss that new services has been added, what do you think?

Reply:
I added a extra [commit](https://github.com/einride/protoc-gen-go-aip-test/pull/267/commits/b31630f642b549272e10ec2e30f6ecd39ec7429a), which makes the service specific test functions private.
With that there is only one way of starting the tests with the new setup, lets discuss it!

Reply:
Nice! I think that makes sense. I think the same as the [discussion above](https://github.com/einride/protoc-gen-go-aip-test/pull/267#discussion_r1796510939) that while it is technically breaking I think we could treat it as a fix 😅 

Won't the `feat!` commit push out a new major version v1? Not sure that is worth it, but it is technically correct 🙃

Reply:
> Nice! I think that makes sense. I think the same as the https://github.com/einride/protoc-gen-go-aip-test/pull/267#discussion_r1796510939 that while it is technically breaking I think we could treat it as a fix 😅

True!

Yeah, I don't want to push a new major version :sweat_smile:

Reply:
@alethenorio said one thing, that if people want to use the service specific tests, we could always make them public in the future, but its harder other way around, so I think its good to start with them as private, and keep the API small.